FirebaseInstallations/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# 1.0.0
+# v1.0.0 -- M62
```

Comment:
just to double check: the uploaded SDK version will be "i:1.0.0", right?

Reply:
yes, the version string in the request is "i:1.0.0"